### PR TITLE
Adding  ui-chassis

### PR DESCRIPTION
**Pull request by davidradl**
Creating a draft pr - so Nigel can do verify testing that the docker and tutorials.

### CODE REVIEW

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml [hunk 1]
```diff
@@ -326,7 +323,7 @@
                                 <configuration>
                                     <nodeVersion>${node.version}</nodeVersion>
                                     <npmVersion>${npm.version}</npmVersion>
-                                    <workingDirectory>tools</workingDirectory>
+                                    <workingDirectory>../ui-chassis-spring/tools</workingDirectory>
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml]: I'm a little wary of creating files outside the directory tree we are working in. Clean may get confused, as may other tools such as javadoc or scans. I would like to look at this in a little more detail when I get a chance (tomorrow) to check out the code.

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml [hunk 2]
```diff
@@ -339,7 +336,7 @@
                                 </goals>
                                 <configuration>
                                     <arguments>install polymer-cli --prefix .</arguments>
-                                    <workingDirectory>tools</workingDirectory>
+                                    <workingDirectory>../ui-chassis-spring/tools</workingDirectory>
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml]: as above

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml [hunk 3]
```diff
@@ -350,7 +347,7 @@
                                 </goals>
                                 <configuration>
                                     <arguments>install ../src/main/static --prefix .</arguments>
-                                    <workingDirectory>tools</workingDirectory>
+                                    <workingDirectory>../ui-chassis-spring/tools</workingDirectory>
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml]: as above

File: open-metadata-resources/open-metadata-deployment/docker/egeria/Dockerfile [hunk 4]
```diff
@@ -31,7 +31,7 @@ COPY --chown=egeria:egeria dist/entrypoint.sh /entrypoint.sh
 ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/server-chassis-spring/${version}/maven-metadata.xml /opt/egeria/server-chassis.xml
 ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/access-services-user-interface/${version}/maven-metadata.xml /opt/egeria/user-interface.xml
 ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/server-chassis-spring/${version}/server-chassis-spring-${version}.jar /opt/egeria/server-chassis-spring.jar
-ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/access-services-user-interface/${version}/access-services-user-interface-${version}.jar /opt/egeria/access-services-user-interface.jar
+ADD --chown=egeria:egeria ${repo}/org/odpi/egeria/ui-chassis-spring/${version}/ui-chassis-spring-${version}.jar /opt/egeria/ui-chassis-spring.jar
```
Comment (planetf1) [inline open-metadata-resources/open-metadata-deployment/docker/egeria/Dockerfile]: :-) looks good by inspection

File: open-metadata-resources/open-metadata-deployment/charts/lab/templates/egeria-ui.yaml [hunk 5]
```diff
@@ -62,7 +62,7 @@ spec:
           command:
             - "/bin/bash"
             - "-c"
-            - "java -jar /opt/egeria/access-services-user-interface.jar"
+            - "java -jar /opt/egeria/ui-chassis-spring.jar"
```
Comment (planetf1) [inline open-metadata-resources/open-metadata-deployment/charts/lab/templates/egeria-ui.yaml]: these all look good by inspection

**Comment by planetf1** [comment]
Built, run.. Inspected docker/helm though couldn't run without changes due to the process in place.
Recommend merging.

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml (hunk 1 above)
Comment (planetf1) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml]: Seems to work ok. let's close for now.

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml (hunk 2 above)
Comment (davidradl) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml]: fixed

**Pull request merged**